cdap-ui/package.json

```diff
@@ -151,6 +151,7 @@
     "finalhandler": "0.4.1",
     "font-awesome": "4.6.3",
     "fuse.js": "2.5.0",
+    "helmet-csp": "^2.7.1",
```

Comment:
lock the version?

Reply:
I think with yarn we don't have to do it. Once yarn generates the yarn.lock file we can control the updates(or silent upgrades) by preventing from checking in yarn.lock file. 

Since we have been doing this will update for consistency.